### DISCUSSION TITLE
Welcome to meshgpt-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/meshgpt-pytorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iliavits** [comment]
Hello,

Thanks for sharing meshgpt-pytorch implementation code. Any chance you would add pretrained models? I would like to use you code to construct meshes that fit object silhouettes.  I thought that I can do it by using a differential renderer to adapt transformer mesh generation by backprojecting the object to its silhouettes. I would like to test the feasibility of this approach without the need to retrain gptmesh and the autoencoder from scratch.
Thank you,
ilia Vitsnudel
